src/shared/utils/cleanText.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * See https://github.com/apostrophecms/sanitize-html#sanitize-html for the
+ * library's configuration
+ */
+import sanitizeHtml, {
+  defaults,
+  simpleTransform,
+  IOptions,
+} from 'sanitize-html';
+
+import styles from './styles/clean-text.module.scss';
+
+// List of tags to remove from the library default accepted set
+const blacklist = new Set(['a']);
```

Comment:
```suggestion
const excludedTags = new Set(['a']);
```
I always confuse `blacklist` and `whitelist` - could we use `excludedTags`?

Reply:
Oh yeah, sounds good. I'd always wondered if these terms had similar bad connotations as the "master" branch naming for example. This avoids this issue altogether 👍🏽